web/src/components/__tests__/netflow-table-header.spec.tsx

```diff
@@ -40,4 +58,18 @@ describe("<NetflowTableHeader />", () => {
     expect(wrapper.find(Tr)).toHaveLength(1);
     expect(wrapper.find(Th)).toHaveLength(reducedColumns.length);
   });
+  it("should call sort function on click", () => {
+    const onSort = jest.fn();
+    const wrapper = mount(
+      <NetflowTableHeaderWrapper
+        onSort={onSort}
+        sortIndex={0}
+        sortDirection={"asc"}
+        columns={ColumnsSample}
+      />
+    );
+    expect(wrapper.find(NetflowTableHeader)).toBeTruthy();
+    wrapper.find("button").at(0).simulate("click");
+    expect(onSort).toHaveBeenCalled();
```

Comment:
Should we check what sort has been requested by the click ?
You can use `toHaveBeenCalledWith` to check onSort arguments

Reply:
Done, thanks !